lib/tests/test_fix.rs

```diff
@@ -357,17 +357,22 @@ fn test_already_fixed_descendant() {
     )
     .unwrap();
 
-    // TODO: Should have fixed both commits
-    assert_eq!(summary.rewrites.len(), 1);
+    assert_eq!(summary.rewrites.len(), 2);
```

Comment:
Optional: this is confusing, I wouldn't expect the second commit to be counted here in this situation. Perhaps a comment would be nice, or just delete some of this and leave the tree test below.

Reply:
The commit is rewritten, so I think that makes sense. But I agree that it makes sense to not consider the second commit fixed. I've updated the code and test. Feel free to take another look.